pybaseball/datasources/fangraphs.py

```diff
@@ -42,16 +43,55 @@ def __init__(self):
+        known_columns = [x for x in columns if x in data.columns]
+        if known_columns == []:
+            return data
+
+        return data.sort_values(columns, ascending=ascending)
+
     def _validate(self, data: pd.DataFrame) -> pd.DataFrame:
         return data
 
-    def fetch(self, start_season: int, end_season: Optional[int] = None, types: FanGraphsStat = None,
+    def fetch(self, start_season: int, end_season: Optional[int] = None, types: List[FanGraphsStat] = [],
               league: FanGraphsLeague = FanGraphsLeague.ALL, qual: Optional[int] = None, split_seasons: bool = True,
               month: FanGraphsMonth = FanGraphsMonth.ALL, on_active_roster: bool = False, minimum_age: int = MIN_AGE,
               maximum_age: int = MAX_AGE, team: str = '0', _filter: str = '', players: str = '',
               position: FanGraphsPositions = FanGraphsPositions.ALL, max_results: int = 1000000,) -> pd.DataFrame:
 
-        types = self.DEFAULT_TYPES if types is None else types
+        """
+        Get leaderboard data from FanGraphs.
+
+        ARGUMENTS:
+        start_season       : int                : First season to return data for
+        end_season         : int                : Last season to return data for
+                                                  Default = start_season
+        types              : str                : The columns of data to return
+        league             : FanGraphsLeague    : League to return data for: ALL, AL, FL, NL
+                                                  Default = FanGraphsLeague.ALL
+        qual               : Optional[int]      : Minimum number of plate appearances to be included.
+                                                  If None is specified, the FanGraphs default ('y') is used.
+                                                  Default = None
+        split_seasons      : bool               : True if you want individual season-level data
+                                                  False if you want aggregate data over all seasons.
+                                                  Default = False
+        month              : FanGraphsMonth     : Month to filter data by. FanGraphsMonth.ALL to not filter by month.
+                                                  Default = FanGraphsMonth.ALL
+        on_active_roster   : bool               : Only include active roster players.
+                                                  Default = False
+        minimum_age        : int                : Minimum player age.
+                                                  Default = 0
+        maximum_age        : int                : Maximum player age.
+                                                  Default = 100
+        team               : str                : Team to filter data by.
+                                                  Specify "0,ts" to get aggregate team data.
+        position           : FanGraphsPositions : Position to filter data by.
```

Comment:
Same re string. Easy to do '2B', rather than go figure out where the enum is

Reply:
Implemented.